website/common/script/content/appearance/chair.js

```diff
@@ -1,11 +1,18 @@
 import prefill from './prefill.js';
 
 export default prefill({
+  /* eslint-disable quote-props */
```

Comment:
I don't think this rule is necessary, the quotes can be removed but then it'll become necessary to disable the `camelcase` rule, also the rule when possible should be disabled only for the necessary lines:

```
...
/* eslint-disable camelcase */
...props not in came case....
/* eslint-enable camelcase */
```

otherwise there's a risk of disabling the rule for more than it's necessary

Reply:
I moved the disable rule comment down below, just above where the quoted section begins.